meson.build

```diff
@@ -2952,6 +2967,22 @@ if conf.get('ENABLE_TMPFILES') == 1
                      # https://github.com/mesonbuild/meson/issues/2681
                      args : exe.full_path())
         endif
+
+        public_programs += executable(
+                'systemd-tmpfiles.standalone',
+                'src/tmpfiles/tmpfiles.c',
+                'src/tmpfiles/offline-passwd.c',
+                'src/tmpfiles/offline-passwd.h',
```

Comment:
I think it'd be better to define a `systemd_tmpfiles_sources` in `src/tmpfiles/meson.build` and reference it here and above, rather than to repeat this list.

Reply:
Done!